tests/ui/intrinsics/not-overridden.stderr

```diff
@@ -0,0 +1,4 @@
+error: linking with `cc` failed: exit status: 1
```

Comment:
This is a pretty hacky way to ensure this. I can probably make it ICE earlier in the compiler, but that needs to be done manually in each backend.